code/src/styles/styles.scss

```diff
@@ -0,0 +1,477 @@
+    font-size: 20px;
+  }
+
+  @include respond(large) {
+    font-size: 25px;
+  }
+
+  &__nav {
+    position: relative;
+
+    &--open {
+      display: flex;
+      flex-direction: column;
+      justify-content: space-evenly;
+      position: absolute;
+      top: 14vh;
+      left: 0;
+      right: 0;
+      z-index: 1;
+      text-align: center;
+      background: $dark-blue;
+      height: 35vh;
+    }
+
+    &--closed {
+      display: none;
+    }
+  }
+
+  &__button {
+    display: inline;
+    border: none;
+    background: transparent;
+    font-family: 'Poiret One', cursive;
+    font-weight: 600;
+    text-transform: uppercase;
+    font-size: 16px;
+    padding: 10px;
+    border-radius: 0px;
+    color: $cream;
+    cursor: pointer;
+
+    &:hover {
+      background: $cream;
+      color: $dark-blue;
+    }
+  }
+
+  &__menu {
+    font-size: 30px;
+    font-weight: 400;
+  }
+
+  &__link-container {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__link {
+    text-decoration: none;
+    color: $cream;
+    font-weight: bold;
+    padding: 10px;
+    width: 100%;
+
+    &:hover {
+      background: $cream;
+      color: $dark-blue;
+    }
+  }
+
+  &__logo {
+    height: 100%;
+    color: $cream;
+    text-decoration: none;
+
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+
+  &__image {
+    height: 100%;
+    filter: invert(1);
+    color: #000;
+  }
+
+  &__name {
+    font-size: 15px;
+    text-align: right;
+  }
+
+  @include respond(medium) {
+    &__nav {
+      display: flex;
+    }
+
+    &__nav--open {
+      all: unset;
+      display: flex;
+    }
+
+    &__nav--closed {
+      display: flex;
+    }
+
+    &__button {
+      display: none;
+    }
+
+    &__name {
+      font-size: 20px;
+    }
+  }
+}
+
+/*--- List Grid ---*/
+
+.list {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  position: relative;
+  width: 100%;
+
+  @include respond(medium) {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
+
+/*--- List Movie Thumb ---*/
+
+.movie-thumb {
+  position: relative;
+  text-align: center;
+
+  &__link {
+    color: $cream;
+    text-decoration: none;
+    font-weight: bold;
+    display: block;
+  }
+
+  &__image {
+    width: 100%;
+  }
+
+  &__overlay {
+    display: none;
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background: linear-gradient(rgba($dark-blue, 0.2) 0, rgb(21, 23, 61) 80%);
+    padding: 40px 10px;
+    transition: all 0.5s ease-in-out;
+  }
+
+  &__title {
+    font-size: 35px;
+    margin-bottom: 10px;
+  }
+
+  &:hover .movie-thumb__overlay {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+}
+
+/*** MOVIE DETAIL PAGE ***/
+
+.detail {
+  display: grid;
+  min-height: 100vh;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  color: $cream;
+
+  &__info-container {
+    display: flex;
+    align-items: flex-end;
+    padding: 5%;
```

Comment:
I have gotten some feedback on using % in padding and margin, and have learned that it's better with pixels.